packages/headless/src/features/commerce/parameters/parameters-serializer.ts

```diff
@@ -254,3 +261,37 @@ export function deserializeSortCriteria(
     };
   }, buildFieldsSortCriterion([]));
 }
+
+function sanitizeNumericFacetParams(params: string) {
+  const seenNumericFacets = new Set<string>();
+  const dedupedNumericFacets = new Set<string>();
+  const sanitized = [];
+
+  const isManualNumericFacet = (param: string) => param.startsWith('mnf-');
+  const isNumericFacet = (param: string) =>
+    param.startsWith('nf-') || isManualNumericFacet(param);
```

Comment:
Let's move this out of the function to avoid reinitializating them every call :)